physics/Interstitials/UFS_SCM_NEPTUNE/sfcsub.F

```diff
@@ -8038,6 +8189,31 @@ subroutine clima(lugb,iy,im,id,ih,fh,len,lsoil,slmskl,slmskw,     &
 !           if (me .eq. 0) write(6,*) 'climatological vegetation',
 !    &                                ' cover read in for mon=',mon
           endif
+!
+!  leaf are index
+!
+          if (fnlaic(1:8) .ne. '        ') then
+            if ( index(fnlaic, "tileX.nc") == 0) then ! grib file
```

Comment:
since there is no grib file, what happens here? what does the model end up with for an LAI?

Reply:
This was added to be consistent with other variables.  If the dveg = 9 and an FNLAIC is defined and pointing to an existing the model will read from file. If the grib file doesn't exist , I think It should result in a file not found error. I will request @GeorgeGayno to respond/confirm please.